### PR TITLE
🚀 re-natal completely removed from the project and replaced with clj-rn

### PR DESCRIPTION
**Pull request by alwx**
In #core-developer we recently decided to get rid of `re-natal`, because we now have an amazing `clj-rn` that can easily be used instead. 

This PR replaces `re-natal` with `clj-rn`.

Status: ready.

### CODE REVIEW

File: scripts/run-osx [hunk 1]
```diff
@@ -60,13 +60,13 @@ device=$(/Applications/Genymotion\ Shell.app/Contents/MacOS/genyshell -c "device
 open -a /Applications/Genymotion.app/Contents/MacOS/player.app --args --vm-name "${device##*| }"
 fi
 
-# Install deps, prepare for genymotion and figwheel
-lein deps && npm install && ./re-natal use-android-device "${device_type}" && ./re-natal use-figwheel && npm install
+# Install deps
+lein deps && npm install
```
Comment (jeluard) [inline scripts/run-osx]: Is the lein bit necessary?

Comment (alwx) [reply scripts/run-osx]: it's likely that we can remove it. I will check it

**Review by dmitryn — APPROVED** [review]

**Comment by dmitryn** [comment]
I wonder how this will play with https://github.com/status-im/status-react/pull/5203 where `re-natal` is still a dependency.

**Comment by alwx** [comment]
@dmitryn interesting question :) From what I see now, it can easily be removed there, so I don't think it's a real problem. I wrote a message there anyway.

**Review by vitvly — APPROVED** [review]

**Comment by vitvly** [comment]
Please correct me if I'm wrong: temporarily, taking into account #5203, clj-rn and re-natal can coexist safely, right? The `prepare-for-platform.sh` script from #5203 would just need to restore `.re-natal` for desktop builds to work, not interfering at all with mobile builds.

**Comment by jeluard** [comment]
@siphiuel Goal is to simplify our builds. What do you use to build desktop? Keeping it for desktop only is definitely an option!

**Comment by vkjr** [comment]
@vlasov, @jeluard, I was thinking about the same. It looks like this PR is ready to be merged. But `clj-rn` doesn't support `desktop` platform yet. So possible way to go is to still build desktop with re-natal while mobile platforms built with `clj-rn` until latest extended.

**Pull request merged**